Makefile

```diff
@@ -1,7 +1,14 @@
+SERVICE := $(or ${SERVICE},quay.io/ocpmetal/ignition-manifests-and-kubeconfig-generate:latest)
+
+
 all: pep8 pylint build
 
 build:
 	skipper build ignition-manifests-and-kubeconfig-generate
 
+update: build
```

Comment:
I, personally against target that builds and pushes. I think it may lead to people pushing, by mistake, their debug images into latest

Reply:
Agree, Although people shouldn't be login to ocpmetal...